jobs/config.json

```diff
@@ -8519,6 +8519,27 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7-upgrade-cluster-init": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.6",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.6",
+      "--kubernetes-anywhere-upgrade-provider=kubeadm",
```

Comment:
We have to update kubetest to allow these CLI args, right?
Then also bump the kubeadm image everywhere to a newer version...

Reply:
@jessicaochen @luxas BTW do we need a new `kubernetes-anywhere-upgrade-provider` arg, or can we use `kubernetes-anywhere-phase2-provider`?

Reply:
I've added the former anyway, just wanted to check